examples/lib/stories/input/keyboard_listener_component_example.dart

```diff
@@ -0,0 +1,131 @@
+import 'package:examples/commons/ember.dart';
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flame/palette.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+class KeyboardListenerComponentExample extends FlameGame
+    with HasKeyboardHandlerComponents {
+  static const String description = '''
+    Similar to the default Keyboard example, but shows a different implementation approach,
+    which uses Flame's KeyboardListenerComponent to handle input.
+    Usage: Use A S D W to steer Ember.
+  ''';
+
+  static final Paint white = BasicPalette.white.paint();
+  static const int speed = 200;
+
+  late final Ember ember;
+  final Vector2 velocity = Vector2(0, 0);
+
+  @override
+  Future<void> onLoad() async {
+    ember = Ember(position: size / 2, size: Vector2.all(100));
+    add(ember);
+
+    add(
+      KeyboardListenerComponent(
+        keyUp: {
+          LogicalKeyboardKey.keyA: (keys) => _handleKey(
+                false,
+                LogicalKeyboardKey.keyA,
+                keys,
+              ),
+          LogicalKeyboardKey.keyD: (keys) => _handleKey(
+                false,
+                LogicalKeyboardKey.keyD,
+                keys,
+              ),
+          LogicalKeyboardKey.keyW: (keys) => _handleKey(
+                false,
+                LogicalKeyboardKey.keyW,
+                keys,
+              ),
+          LogicalKeyboardKey.keyS: (keys) => _handleKey(
+                false,
+                LogicalKeyboardKey.keyS,
+                keys,
+              ),
+        },
```

Comment:
can be a followup, but I wonder if we can provide a less verbose API where you just list the keys we want to listen on an array and bind 1 `_handleKey` function taking 3 parameters